src/form.js

```diff
@@ -353,6 +353,16 @@ define([
                 return m.p.nodeID === nodeID;
             });
         },
+        getMugChildrenByChoiceValue: function (mug, nodeID) {
```

Comment:
Would probably be clearer to rename `nodeID` to `value` since it's not a node ID.